src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -50,6 +54,28 @@ public void verifyCategoryTransitionThroughDropdownTest(String title) {
                 .as("Product should correspond " + title + " category")
                 .contains(title);
     }
+
+    @DataProvider
+    public Object[][] rozetkaSocialMedia() {
+        return new Object[][]{
+                {"Facebook"},
+                {"Telegram"}
+        };
+    }
+
+    @Test(dataProvider = "rozetkaSocialMedia")
+    public void verifySocialMediaLinks(String socialMediaName) {
+        homePage.openSocialMediaPage(socialMediaName);
+        var expectedUrl = SocialMedia.valueOf(socialMediaName.toUpperCase()).getSocialMediaLink();
+        BrowserTabHelper.switchToTab(1);
+        var actualUrl = WebDriverRunner.getWebDriver().getCurrentUrl();
+        BrowserTabHelper.closeTab();
```

Comment:
same, and probably move this method under assert, it is needed in case if assert fails and on this tab should make a screenshot